[2025-03-02T06:46:28.966+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-02T06:46:28.985+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: redshift_to_s3_dag.upload_to_s3 manual__2025-03-02T06:46:23.459993+00:00 [queued]>
[2025-03-02T06:46:28.994+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: redshift_to_s3_dag.upload_to_s3 manual__2025-03-02T06:46:23.459993+00:00 [queued]>
[2025-03-02T06:46:28.995+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-03-02T06:46:29.006+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): upload_to_s3> on 2025-03-02 06:46:23.459993+00:00
[2025-03-02T06:46:29.013+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=899) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-03-02T06:46:29.014+0000] {standard_task_runner.py:72} INFO - Started process 901 to run task
[2025-03-02T06:46:29.013+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'redshift_to_s3_dag', 'upload_to_s3', 'manual__2025-03-02T06:46:23.459993+00:00', '--job-id', '146', '--raw', '--subdir', 'DAGS_FOLDER/final_aggregation_dag.py', '--cfg-path', '/tmp/tmpa5nvl7x_']
[2025-03-02T06:46:29.015+0000] {standard_task_runner.py:105} INFO - Job 146: Subtask upload_to_s3
[2025-03-02T06:46:29.031+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:209 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-03-02T06:46:29.066+0000] {task_command.py:467} INFO - Running <TaskInstance: redshift_to_s3_dag.upload_to_s3 manual__2025-03-02T06:46:23.459993+00:00 [running]> on host 763fac820cd7
[2025-03-02T06:46:29.151+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='redshift_to_s3_dag' AIRFLOW_CTX_TASK_ID='upload_to_s3' AIRFLOW_CTX_EXECUTION_DATE='2025-03-02T06:46:23.459993+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-02T06:46:23.459993+00:00'
[2025-03-02T06:46:29.152+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-02T06:46:29.190+0000] {credentials.py:1278} INFO - Found credentials in shared credentials file: ~/.aws/credentials
[2025-03-02T06:46:30.468+0000] {logging_mixin.py:190} INFO - Uploaded /tmp/total_revenue_by_category.csv to s3://finalresultsevents/final_metrics_result/total_revenue_by_category.csv
[2025-03-02T06:46:30.470+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-03-02T06:46:30.482+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-02T06:46:30.483+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=redshift_to_s3_dag, task_id=upload_to_s3, run_id=manual__2025-03-02T06:46:23.459993+00:00, execution_date=20250302T064623, start_date=20250302T064628, end_date=20250302T064630
[2025-03-02T06:46:30.549+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-02T06:46:30.566+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-03-02T06:46:30.567+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
